test/testdata/gci.go

```diff
@@ -0,0 +1,17 @@
+//args: -Egoimports
+//config: linters-settings.goimports.local-prefixes=github.com/golangci/golangci-lint
+package goimports
+
+import (
```

Comment:
will be great if we can have negative test case as well

Reply:
I see dirs under `test/testdata` while for errcheck it has some test data files, so i can do like errcheck does?